process_ocaml/base/ProcessCmt.re

```diff
@@ -204,13 +222,17 @@ and forSignatureType = (env, signature) => {
 
   {Module.exported, topLevel}
 } and forModuleType = (env, moduleType) => switch moduleType {
-  | Types.Mty_ident(path) => Module.Ident(path)
+  | Types.Mty_ident(path) => Module.Ident(Shared.mapOldPath(path))
 #if 402
   | Mty_alias(path) =>
 #else
   | Mty_alias(_ /* 402*/, path) =>
 #endif
+#if 407 
```

Comment:
What if, instead of having these `#if 407`s everywhere, we made `Shared.mapOldPath` have the `#if` inside it?